components/nimbus/build.gradle

```diff
@@ -0,0 +1,150 @@
+// =============================================
+// We're doing something a bit weird here, so
+// READ THIS CAREFULLY before editing this file.
+// =============================================
+//
+// This is a lightly-modified version of the `build.gradle` file from Nimbus,
+// which you can find at `../external/nimbus-sdk/android/build.gradle`.
+//
+// The aim here is to pre-package the Nimbus Kotlin code and publish it under
+// `org.mozilla.appservices:nimbus` rather than `org.mozilla.experiments:nimbus`,
+// and to make it depend on `org.mozilla.appservices:full-megazord` for the
+// compiled Rust code.
+//
+// We expect consumers that want to use Nimbus and AppServices together, will
+// use our re-packaged version rather than standalone Nimbus SDK.
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+
+android {
+    ndkVersion rootProject.ext.build.ndkVersion
+    compileSdkVersion rootProject.ext.build.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.build['minSdkVersion']
+        targetSdkVersion rootProject.ext.build['targetSdkVersion']
+
+        buildConfigField("String", "JNA_LIBRARY_NAME", "\"megazord\"")
```

Comment:
TODO: we'll need to add something like https://github.com/mozilla/glean/pull/1290 in Nimbus as well in order for this to work.

Reply:
For Nimbus, this is currently taken care of in the android-components wrapper, which sets a system property to tell Nimbus where to look for its shared library.